data-prepper-logstash-configuration/src/test/java/org/opensearch/dataprepper/logstash/mapping/DateLogstashPluginAttributesMapperTest.java

```diff
@@ -63,7 +63,7 @@ void convert_match_attribute_with_single_pattern_from_list_to_date_match_list_te
                 Collections.singletonMap(LOGSTASH_DATE_MATCH_ATTRIBUTE_NAME, dataPrepperMatchAttribute));
 
         final Map<String, Object> actualPluginSettings =
-                dateLogstashPluginAttributesMapper.mapAttributes(Collections.singletonList(dateMatchAttribute), mappings);
+                dateLogstashPluginAttributesMapper.mapAttributes(Collections.singletonList(dateMatchAttribute), mappings).get(0).getPluginSettings();
```

Comment:
Please split all these usages of `get(0).getPluginSettings()`. If you have a null pointer in here, it can be hard to track down.

```
List<PluginModel> plugins = dateLogstashPluginAttributesMapper.mapAttributes(Collections.singletonList(dateMatchAttribute), mappings);
assertThat(plugins, notNullValue());
assertThat(plugins.size(), equalTo(1));
...
```